### PR TITLE
"Presentation" mode 

### PR DESCRIPTION
**Pull request by wkpn**
Covers #98 

Allows customer to hide side menu and user card with a single click.
Icon is a subject to change

### CODE REVIEW

**Review by skalitin — APPROVED** [review]
Looks good! Also I'd add a small tooltip to the 'toggle widescreen' icon )

File: webui/src/store/modules/user.js [hunk 1]
```diff
@@ -50,6 +53,9 @@ export default {
     },
 
     mutations: {
+        CHANGE_VIEW_MODE(state){
+          state.widescreen = !state.widescreen;
```
Comment (denis1stomin) [inline webui/src/store/modules/user.js]: Also it would be nice to use the same code offset if you don't mind :)

**Comment by denis1stomin** [comment]
And one more thing ;) Can you plz move it to the left of username?)

**Comment by wkpn** [comment]
@skalitin @denis1stomin updated

**Comment by denis1stomin** [comment]
May be call it just "Presentation Mode"?
Arguments :)
1. You always see "Enter" even when the mode is activated
2. In my opinion widescreen doesn't mean that some info should be hidden. For me it means that the same info will be shown but as widescreen. In the case of "presentation mode" it sounds like something related to info content will be changed.
Just IMHO :)

File: webui/src/styles/main.less [hunk 2]
```diff
@@ -61,7 +61,20 @@ a {
       .saving-indicator {
         margin-right: 30px;
       }
+      .widescreen {
+        margin-left: 45px;
```
Comment (denis1stomin) [inline webui/src/styles/main.less]: It would be nice to change 
margin-left: 45px;   ->   margin-right: 7px;

File: webui/src/styles/main.less [hunk 3]
```diff
@@ -61,7 +61,20 @@ a {
...
+          vertical-align: top;
+          fill: white;
+          transition: all 0.3s ease-in;
+          &:hover {
+            transform: scale(1.5);
+          }
+        }
+      }
       .log-out {
         margin-left: 15px;
```
Comment (denis1stomin) [inline webui/src/styles/main.less]: And here margin-left: 7px;

Comment (wkpn) [reply webui/src/styles/main.less]: updated

**Review by denis1stomin — APPROVED** [review]
Thank you very much! :smiley:

**Pull request merged**